services/coolifyService.js

```diff
@@ -359,9 +387,10 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_curr = name.allReplace(coolifyDicts.curr);
 		let cool_name_fairy = name.allReplace(coolifyDicts.fairy);
 		let cool_name_dot = name.allReplace(coolifyDicts.dot);
-    let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
+    	let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
```

Comment:
Please fix the indentation.

Reply:
Fixed indentation issue.https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/14caec93d55eafd1e045b65162ad48312bf31377